manifests/Cromha Expansion.yaml

```diff
@@ -0,0 +1,14 @@
+name: Cromha Expansion Plugin
+authors: @OcelotWalrus/Cromha, @Lumbar527
```

Comment:
```suggestion
authors: "@OcelotWalrus/Cromha, @Lumbar527"
```
Needs some form of quotes to not break YAML syntax

Reply:
Done